neqo-common/src/qlog.rs

```diff
@@ -144,3 +149,40 @@ pub fn new_trace(role: Role) -> qlog::TraceSeq {
         }),
     }
 }
+
+#[cfg(test)]
+mod test {
+    use const_format::concatcp;
+    use qlog::events::Event;
+    use test_fixture::{new_neqo_qlog, EXPECTED_LOG_HEADER};
+
+    const EV_DATA: qlog::events::EventData =
+        qlog::events::EventData::SpinBitUpdated(qlog::events::connectivity::SpinBitUpdated {
+            state: true,
+        });
+
+    const EXPECTED_LOG_EVENT: &str = concatcp!(
+        "\u{1e}",
+        r#"{"time":0.0,"name":"connectivity:spin_bit_updated","data":{"state":true}}"#,
+        "\n"
```

Comment:
Any reason that [`concat!()`](https://doc.rust-lang.org/std/macro.concat.html) is insufficient here?